pkg/manager/cli_download_install.go

```diff
@@ -0,0 +1,319 @@
+package manager
+
+import (
+	"context"
+	"errors"
+	"sort"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func EnableHypershiftCLIDownload(hubclient client.Client, log logr.Logger) error {
+	// get the most current version of MCE CSV from multicluster-engine namespace
+	csv, err := GetMCECSV(hubclient, log)
+	if err != nil {
+		log.Error(err, "failed to get the most current version of MCE CSV from multicluster-engine namespace")
```

Comment:
minor nit: `most recent` or `current` but not `most current`

Reply:
Yes `current`